tests/Nox.Solution.Tests/ValidationTests.cs

```diff
@@ -13,4 +22,30 @@ public void Environment_names_are_unique()
 
         Assert.Equal("The environment name 'test' is duplicated. Environment names must be unique in a solution.", ex.Errors.First().ErrorMessage);
     }
+
+    [Fact]
+    public void Solution_DeserializeWithValidation_ShoouldThrow()
```

Comment:
a typo on Shouuld...

I would suggest the test name: Deserialize_With_NoxYamlSerializer_ThrowsException, what do y think? Scenario, Method, Expected Result

```suggestion
    public void Deserialize_With_NoxYamlSerializer_ThrowsException()
```

Reply:
Agree but without extra underline Deserialize_WithNoxYamlSerializer_ThrowsException